ts/annotations/annotations.src.ts

```diff
@@ -1198,13 +1219,29 @@ merge(
         },
 
         render: function (this: Highcharts.Annotation): void {
-            var renderer = this.chart.renderer;
+            var renderer = this.chart.renderer,
+                plotOptions = this.chart.options.plotOptions as any,
+                defer = this.options.defer;
+
+            if (defer) {
+                // If defer duration is set use set value,
+                // else inherits from animation set in plotOptions
+                this.options.defer = isObject(this.options.defer) ? this.options.defer : (
+                    plotOptions.series && defined(plotOptions.series.animation) ?
+                        plotOptions.series.animation :
+                        plotOptions.line.animation
+                );
+                if (this.chart.renderer.forExport) {
+                    this.options.defer = false;
+                }
+            }
```

Comment:
Please, move this to `annotation.init` method. Then store this logic in `this.defer` property so we can easily access this later (e.g. for exporting) instead of modifying `this.options.defer`.

Reply:
@bre1470 - can we define `this.defer` as `false|AnimationOptionsObject`? So later it can be simply written as `if (this.defer) { ..animate(defer) }` ? Any cons?

Reply:
Yes, that would improve the situation with the typing, but then it needs to be changed in the options as well (doclets & interfaces).

Reply:
@bre1470 Should the mentioned option `this.defer` be introduce in doclets syntax too? It is an internal property which is defined by `options.defer` (which user is able to control) and other conditions.

Reply:
No, I do not think so. This was a comment about the type used in API options doclets, but as I see `true` is there in use to activate defaults. So nothing has to be changed. Sorry for the confusion.